resources/Components/DisplayValue/index.js

```diff
@@ -0,0 +1,82 @@
+import React from 'react'
+import { displayValueData } from '../../utils/displayValue'
+
+const Approx = ({ approximationSymbol }) => {
+  return approximationSymbol && (
```

Comment:
in general it's good practice to check for undefined _before_ passing into a function/component, keeps the logic in the components simple and doing this check at a higher level prevents us from having to do the check in every individual sub-component

Reply:
Going to tag @floating here for reference of the above - these were his changes, but I will update this now

Reply:
Also I probably should have caught this in review.  Anyway